src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.html

```diff
@@ -1,350 +0,0 @@
-      <div
-        data-test-subj="toggleYAxisOptions-{{axis.id}}"
-        class="visEditorSidebar__collapsibleTitleLabel"
-        ng-click="isValueAxisOpen = !isValueAxisOpen"
-        kbn-accessible-click
-        aria-expanded="{{!!isValueAxisOpen}}"
-        aria-controls="axisOptions{{axis.id}}"
-        aria-label="Toggle {{axis.name}} options"
-      >
-        <icon class="kuiSideBarOptionsLink__caret" color="'primary'" aria-hidden="true" ng-if="isValueAxisOpen" type="'arrowDown'" size="'s'"></icon>
-        <icon class="kuiSideBarOptionsLink__caret" color="'primary'" aria-hidden="true" ng-if="!isValueAxisOpen" type="'arrowRight'" size="'s'"></icon>
-
-        <span class="visEditorSidebar__collapsibleTitleText">
-          {{axis.name}}
-        </span>
-      </div>
-
-      <div class="visEditorSidebar__collapsibleTitleDescription" tooltip="{{getSeries(axis)}}">{{getSeriesShort(axis)}}</div>
-
-      <button
-        ng-hide="editorState.params.valueAxes.length === 1"
-        aria-label="Remove Y axis"
-        ng-click="removeValueAxis(axis)"
-        tooltip="Remove Y axis"
-        tooltip-append-to-body="true"
-        type="button"
-      >
-        <icon color="'danger'" aria-hidden="true" type="'cross'" size="'m'"></icon>
-      </button>
-    </div>
-
-    <div id="axisOptions{{axis.id}}" ng-show="isValueAxisOpen">
-      <!-- General -->
-
-      <div class="visEditorSidebar__formRow">
-        <label
-          class="visEditorSidebar__formLabel"
-          for="{ 'valueAxisShow' + $index }"
-          i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.showLabel"
-          i18n-default-message="Show"
-        ></label>
-        <div class="visEditorSidebar__formControl">
-          <input class="kuiCheckBox" id="{ 'valueAxisShow' + $index }" type="checkbox" ng-model="axis.show">
-        </div>
-      </div>
-
-      <div class="visEditorSidebar__formRow">
-        <label
-          class="visEditorSidebar__formLabel"
-          for="{{ 'valueAxisTitle' + $index }}"
-          i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.titleLabel"
-          i18n-default-message="Title"
-        ></label>
-        <div class="visEditorSidebar__formControl">
-          <input
-            id="{{ 'valueAxisTitle' + $index }}"
-            class="kuiInput visEditorSidebar__input"
-            type="text"
-            ng-model="axis.title.text"
-          >
-        </div>
-      </div>
-
-      <div class="visEditorSidebar__formRow">
-        <label
-          class="visEditorSidebar__formLabel"
-          for="{{ 'valueAxisPosition' + $index }}"
-          i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.positionLabel"
-          i18n-default-message="Position"
-        ></label>
-        <div class="visEditorSidebar__formControl">
-          <select
-            id="{{ 'valueAxisPosition' + $index }}"
-            class="kuiSelect visEditorSidebar__select"
-            ng-change="updateAxisName(axis)"
```

Comment:
**note:** moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-f3cb1c2eb82ad19bdb7b7cda32e0942bR101